---
layout: collection
title: About me
permalink: /about/
author_profile: true
read_time: true
comments: # true
share: true
related: false
header:
    image: /assets/images/Lake_Tahoe.jpg
---

During my years as a student of Science and Mathematics during my [High School](https://childrens-academy.co.in/) years back in India, Biology interested me apart from Computers. The first aspect of Biology that interested me was Genetics. It was all so interesting and fascinating and at that point of time, and I never knew that this fascination of mine towards both biology and computers will drive me towards Bioinformatics, the supposed jargon at that time. It was through internet that I got to know about Bioinformatics and its application. Those made me pretty much sure about my future path. I started reading about the Human Genome project, g_enes_, genomes and it all started attracting me.

What better a platform would I have got at that time than pursuing a Bachelor’s degree in Bioinformatics at [D Y Patil University](http://www.dypatil.edu/schools/biotech-and-bioinformatics/), which not only made me aware of different facets of bioinformatics but also strengthened my Mathematics and Computer Science skills. The first computer language I learnt was C and it really changed my approach towards programming. Though a newbie at programming at that time, I always had friends with whom I would discuss better programming practices.

It was during the sophomore year we went for my first curriculum training at [Institute of Bioinformatics and Applied Biotechnology](http://www.ibab.ac.in/), Bengaluru, India on GENES to DRUGS: In-Silico Drug Discovery, It was a 25 days training where we had a hands on experience on [SGI machines using IRIX](https://en.wikipedia.org/wiki/IRIX) with different tools related to bioinformatics such as GCG, Insight II, Cerius 2 and Catalyst all part of the [Accelrys](https://en.wikipedia.org/wiki/Accelrys) suite for [molecular modelling](https://en.wikipedia.org/wiki/Molecular_modelling). Here we came face to face with computational biology an aspect of bioinformatics that we thought was all limited to [Rasmol](http://www.openrasmol.org/), [Blast](https://blast.ncbi.nlm.nih.gov/Blast.cgi), [Fasta](https://www.ebi.ac.uk/Tools/sss/fasta/), [Homology modeling](https://en.wikipedia.org/wiki/Homology_modeling), searching [Protein Data Bank](https://www.rcsb.org/pdb/home/home.do), [Pfam](http://pfam.xfam.org/) etc. Later during the same we wrote [Perl](http://perl.org) programs, accessed [Biological Databases](https://academic.oup.com/nar/issue/45/D1), worked on various bioinformatics tools and developed small applications using [Perl](http://perl.org). We learned [MS Access](https://en.wikipedia.org/wiki/Microsoft_Access) through a project for our course on Airlines Reservation system. It was all exciting, not only developing my skills in wet-lab techniques like DNA extraction, developing cell media, cell plating and many more but also sharpening my programming skills.

While doing my senior design project at [Bhabha Atomic Research Center (BARC)](http://www.barc.gov.in/), Mumbai, India, we had an opportunity to work with senior scientist which explained me the very importance of understanding the basics for whatever you learn, at BARC we did a wet lab project on Characterization of Fusarium Monoliforme, which we combined with bioinformatics by developing [C++](http://www.cplusplus.com/) program to identify optimal fungal growth determined by the data obtained from the experimental results. By then, I had decided that I want to learn more in this field and, that this is just the start. I started looking for graduate courses in India but could find no satisfactory courses at all and I decided to fly to United States of America for my higher studies.

Important attributes of a successful data scientist include the ability to advance human knowledge, understanding by providing meaningful insights from the coarse data available at hand. I was fortunate to imbibe these traits in the department of [Bioinformatics and Computational Biology](http://ssb.gmu.edu/) at [George Mason University](http://gmu.edu). My Master’s program was a great learning curve. Not only I expanded my interests into computational biology but also towards software engineering component of the field. From beinginvolved in projects for building a database of [Human Genes with HUGO Gene Nomenclature](https://www.genenames.org/) using [MySQL](https://www.mysql.com/) and providing it a web interface for searching it using [Perl](http://perl.org) and [CGI](http://perldoc.perl.org/CGI.html). Doing [protein-protein structure alignment](http://rg/sms2/pairwise_align_protein.html) using [Python](https://www.python.org/) and doing a survey of different available [short read sequence assemblers](https://en.wikipedia.org/wiki/Sequence_assembly) with respect to the data produced by the sequencing technology at that time which was from [Solexa](https://en.wikipedia.org/wiki/DNA_sequencing#Illumina_.28Solexa.29_sequencing). We worked on [SVM](https://en.wikipedia.org/wiki/Support_vector_machine) based method (kernel) to assess the reliability of protein-protein interactions. We learned R for statistics through our course of Research Methods. We also worked on Bovine dataset to understand about the variation within a given species and analysis using SNP Genotypes for determining the linkage disequilibrium, haplotypes, and signatures of selection. We helped to design the database for Bovine Hapmap Project and creating a web based program for displaying various data types by querying the database. For master’s project we worked on evolutionary analysis of dopamine d2 receptors, particularly looking at the intra-cellular loop via multiple sequence alignment and creating and comparing phylogentic trees for the differences in evolution using [TOPD/FMTS](http://ppuigbo.me/programs/topd/) and [Rate4Site](https://www.tau.ac.il/~itaymay/cp/rate4site.html).

After my Master’s in Bioinformatics and Computational Biology, I identified my research interest in broad filed of [Genomics](https://en.wikipedia.org/wiki/Genomics). First working as a contractor for [Monsanto Co](https://monsanto.com/), I helped developing frameworks to analyze sequencing data generated from different sequencing technologies, performing genome assembly, working towards making draft genomes, then working as Computational Biologist for [Memorial Sloan Kettering Cancer Center](https://www.mskcc.org/), doing analysis of cancer specific sequencing data to gain meaningful insight in patients tumor molecular profile. As independent research analyst for the past five years, much of my focus has been to do develop, automate and improve the framework to do in depth analysis of the targeted sequencing data produced by [Illumina](https://www.illumina.com/) based machines for clinical as well as research projects, leading to publications ([Google Scholar](https://scholar.google.com/citations?user=ItA13bYAAAAJ&hl=en)) in various aspect of Cancer Genomics research. In addition to a background in research, I have sought out leadership and management experience. For the past two years I have grown to appreciate my role both as mentor and collaborator while advising high school students through their summer internship projects.

As I start my new role as Director of Bioinformatics at [Northwell Health](https://www.northwell.edu/) (NH) working in [Feinstein Institute for Medical Research](http://www.feinsteininstitute.org/) (FIMR), I will be leading the efforts to develop analysis platform for production scale genomics.
